server/sonar-server/src/main/java/org/sonar/server/platform/ServerIdGenerator.java

```diff
@@ -70,7 +72,20 @@ public String generate(String organisationName, String ipAddress) {
     return id;
   }
 
-  boolean isValidOrganizationName(String organisation) {
+  public String generateOrFail(String organizationName, String ipAddress) {
```

Comment:
I don't understand this method : there's already a generate() method.
The code written should be moved to the ws action.